pyppeteer/lifecycle_watcher.py

```diff
@@ -174,3 +174,8 @@ def dispose(self) -> None:
                 fut.cancel()
             except AttributeError:
                 continue
+
+
+def safe_future_set_result(fut: Future, res: Any):
```

Comment:
Maybe we should move it to `helpers.py` to be consistent

Reply:
Maybe. There's quite a few functions that are defined on a module level because they are only used within that module though, which has it's own benefits. eg `collectInterstingNodes`, `serializeTree` in `accessibility.py`, `convertToDisjointRanges` in `coverage.py`, `convertPrintParameterToInches` in `page.py`, etc. I think it makes the most sense to keep functions used exclusively in the module to be defined within it. What are your thoughts?

Reply:
Sidenote: didn't mean for this change to be in this branch, my bad

Reply:
I think it's fine - it's a private function we can always refactor it to where ever if other modules adopt it.